Client/Frontend/Browser/TabToolbar.swift

```diff
@@ -163,14 +141,31 @@ open class TabToolbarHelper: NSObject {
 }
 
 class ToolbarButton: UIButton {
-    var selectedTintcolor: UIColor!
+    static let Themes: [String: Theme] = {
+        var themes = [String: Theme]()
+        var theme = Theme()
+        theme.buttonTintColor = UIConstants.PrivateModeActionButtonTintColor
+        theme.highlightButtonColor = UIColor(rgb: 0xAC39FF)
+        theme.disabledButtonColor = UIColor.gray
+        themes[Theme.PrivateMode] = theme
+        
+        theme = Theme()
+        theme.buttonTintColor = UIColor(rgb: 0x272727)
+        theme.highlightButtonColor = UIColor(rgb: 0x00A2FE)
```

Comment:
Consider putting the hex values into UIConstants. My assumption is that helps ensure color consistency across views by encouraging color definition reuse.

Reply:
I'm leaving color consolidation till the end. There is a separate bug to track this. Just because it'll give me a better understanding of what colors are used where and then I can represent them in UIConstants correctly.